src/components/Menu.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { withRouter } from 'react-router';
 
 const pages = [
-   { title: 'Home', path: '/' },
+   { title: 'Home', path: '/home' },
    { title: 'Map', path: '/map' }
```

Comment:
Something to look into later, the Router is not redirecting to /home if I set the Home's link in `pages` to `/`, I have to have it explicitly set to `/home`.